types/tsconfig.json

```diff
@@ -8,6 +8,6 @@
     "strictFunctionTypes": true,
     "noEmit": true,
     "baseUrl": ".",
-    "paths": { "express-graphql": ["."] }
+    "paths": { "express-graphql": ["./index"] }
```

Comment:
Why did you change that?

Reply:
Dangit, sorry missed that

Reply:
@danielrearden No problem, it happens, e.g. yesterday I committed `console.log` into `graphql-js` 🤦